dot/parachain/network-bridge/wire_message.go

```diff
@@ -67,9 +68,13 @@ func (mvdt WireMessage) ValueAt(index uint) (value any, err error) {
 	return nil, scale.ErrUnknownVaryingDataTypeValue
 }
 
+func (w *WireMessage) SetType(messageType network.MessageType) {
+	w.messageType = messageType
+}
+
 func (w WireMessage) Type() network.MessageType {
 	// TODO: create a wire message type and return that #4108
```

Comment:
I think this TODO is resolved now, right?

Reply:
```suggestion
```

Reply:
yes, it is